bundles/org.openhab.ui/web/src/components/app.vue

```diff
@@ -24,77 +24,81 @@
-                            :class="{ currentsection: currentUrl.indexOf('/settings/scenes') >= 0 }">
+                            :class="{ currentsection: currentUrl.indexOf('/settings/scenes') === 0 }">
                 <f7-icon slot="media" f7="film" color="gray" />
               </f7-list-item>
               <f7-list-item v-if="$store.getters.apiEndpoint('rules')" link="/settings/scripts/" title="Scripts" view=".view-main" panel-close :animate="false" no-chevron
-                            :class="{ currentsection: currentUrl.indexOf('/settings/scripts') >= 0 }">
+                            :class="{ currentsection: currentUrl.indexOf('/settings/scripts') === 0 }">
                 <f7-icon slot="media" f7="doc_plaintext" color="gray" />
               </f7-list-item>
               <f7-list-item v-if="$store.getters.apiEndpoint('rules')" link="/settings/schedule/" title="Schedule" view=".view-main" panel-close :animate="false" no-chevron
-                            :class="{ currentsection: currentUrl.indexOf('/settings/schedule') >= 0 }">
+                            :class="{ currentsection: currentUrl.indexOf('/settings/schedule') === 0 }">
                 <f7-icon slot="media" f7="calendar" color="gray" />
               </f7-list-item>
             </ul>
           </li>
 
-          <f7-list-item link="/settings/addons/" :title="$t('sidebar.addOnStore')" panel-close :animate="false"
-                        :class="{ currentsection: currentUrl.indexOf('/settings/addons/') >= 0 }">
-            <f7-icon slot="media" ios="f7:bag" aurora="f7:bag" md="material:shopping_bag" color="gray" />
+          <!-- Add-on Store -->
+          <f7-list-item link="/addons/" :title="$t('sidebar.addOnStore')" panel-close :animate="false"
+                        :class="{ currentsection: currentUrl.indexOf('/addons/') === 0 }">
+            <f7-icon slot="media" ios="f7:bag_fill" aurora="f7:bag_fill" md="material:shopping_bag" color="gray" />
           </f7-list-item>
 
+          <!-- Developer Tools -->
           <f7-list-item link="/developer/" :title="$t('sidebar.developerTools')" panel-close :animate="false"
-                        :class="{ currentsection: currentUrl.indexOf('/developer/') >= 0 && currentUrl.indexOf('/developer/widgets') < 0 &&
+                        :class="{ currentsection: currentUrl.indexOf('/developer/') === 0 && currentUrl.indexOf('/developer/widgets') < 0 &&
                           currentUrl.indexOf('/developer/blocks') < 0 && currentUrl.indexOf('/developer/api-explorer') < 0 }">
-            <f7-icon slot="media" ios="f7:exclamationmark_shield_fill" aurora="f7:exclamationmark_shield_fill" md="material:extension" color="gray" />
+            <f7-icon slot="media" ios="f7:wrench_fill" aurora="f7:wrench_fill" md="material:extension" color="gray" />
+            <f7-icon slot="media" ios="f7:hammer_fill" aurora="f7:hammer_fill" md="material:extension" color="gray" />
```

Comment:
Thinking we'll have to make a choice :sweat_smile: 

![image](https://github.com/openhab/openhab-webui/assets/2004147/93ad015c-f5c5-4103-82e7-7f5a3c3da254)

Reply:
Good catch, that actually should not have made it into one of my commits.

For Material design, I like `construction` the most (crossed hammer and wrench), from Framework7 I would chose either hammer or wrench. I would even consider to use the Material icon for all:

<img width="179" alt="image" src="https://github.com/openhab/openhab-webui/assets/73423173/dfed0380-bdd2-494a-a45e-1ef4ae25a5f1">

Reply:
I like this icon, it could work.
My initial choice of the shield with the exclamation icon was because the developer tools were initially supposed to be tools that might do harm if you don't know what you're doing, but I agree it's time for a change.

Reply:
I chose this icon for Material, and the wrench from Framework7 for iOS and Aurora. The Material icon IMO does not really match to the other non-Material icons-